### PR TITLE
Update version history for 5.1.1

### PR DESCRIPTION
**Pull request by melissalinkert**
A few of the really minor NPE-type PRs were omitted, but otherwise this should cover everything that is merged, open, or planned before tagging.  Happy to add to it if it looks like anything noteworthy is missing though.

### CODE REVIEW

File: docs/sphinx/about/whats-new.txt [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+
+* Add TIFF writing support to the native C++ implementation
+* Fixed remaining functional differences between Windows and Mac/Linux
+* Improved performance of ImageJ plugin when working with ROIs
+* TIFF export: switch to BigTIFF if .tf2, .tf8, or .btf extensions are used
+* Many bug fixes, including:
+    - fixed upgrade checking to more accurately report when a new version is
+      available
+    - Zeiss CZI
+      - fixed ordering of multiposition data
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: This third level of bullet points is not indented enough - compare the entry below.

**Comment by hflynn** [comment]
Formatting issue shown up on https://www.openmicroscopy.org/site/support/bio-formats5.1-staging/about/whats-new.html
Otherwise, looks fine to me but I leave others to comment on whether there is technical content missing.

**Comment by melissalinkert** [comment]
Formatting fix pushed.

**Comment by hflynn** [comment]
:+1:

**Comment by melissalinkert** [comment]
--no-rebase

**Pull request merged**